src/app/legacy/components/Promo/index.test.jsx

```diff
@@ -35,6 +36,9 @@ const FixtureProgrammes = ({ useLargeImages = false }) => (
   </Promo>
 );
 
+const stringTimestamp = '2023-11-03T05:17:09.393Z';
+const unixTimestamp = 1698995115000;
```

Comment:
Let's move these variables to within the `Promo component - Timestamp` describe - just to keep them all together.

Also, we will have to make these timestamps relative to now, otherwise the tests will fail as time ticks on.